snap/snapcraft.yaml

```diff
@@ -32,40 +32,27 @@ parts:
       
   zap:
     plugin: nil
-    source: https://github.com/project-chip/zap.git
-    source-depth: 1
-    source-tag: v2023.05.04
     build-environment:
-      - NODE_VERSION: v18.16.1
+      - ZAP_VERSION: v2023.09.28
     build-packages:
-      - build-essential 
-      - libpango1.0-dev 
-      - libjpeg-dev 
-      - libgif-dev 
-      - librsvg2-dev
       - wget
+      - unzip
     override-build: |
       if [[ $SNAP_ARCH == "arm64" ]]; then
-        # Install node+npm
-        NODE=node-$NODE_VERSION-linux-$SNAP_ARCH
-        wget https://nodejs.org/dist/$NODE_VERSION/$NODE.tar.xz
-        tar -xJf $NODE.tar.xz
-        export PATH=$PWD/$NODE/bin:$PATH
+        # Download and unzip the prebuilt ZAP (Zigbee Cluster Library configuration tool and generator) binary for the ARM64 architecture
+        wget --no-verbose https://github.com/project-chip/zap/releases/download/$ZAP_VERSION/zap-linux-$SNAP_ARCH.zip
+        unzip -o zap-linux-$SNAP_ARCH.zip
 
-        # Install zap
-        npm install
-        
         # Define the environment needed for the app build
-        echo "export PATH=$PWD/$NODE/bin:$PATH" > env
-        echo "export ZAP_DEVELOPMENT_PATH=$PWD" >> env
-        cat env
+        echo "CRAFT_PART_BUILD=$CRAFT_PART_BUILD"
+        echo "export ZAP_INSTALL_PATH=$CRAFT_PART_BUILD" >> env
```

Comment:
Is there a specific reason for changing lines 60-61? 
$CRAFT_PART_BUILD is the current directory, equivalent to $PWD here. Using the craft variable just makes it harder to understand the code.

Reply:
It looks like some leftover code from debugging and git rebase. Removed.